br/pkg/task/backup.go

```diff
@@ -150,10 +156,19 @@ func (cfg *BackupConfig) ParseFromFlags(flags *pflag.FlagSet) error {
 	if err != nil {
 		return errors.Trace(err)
 	}
+	cfg.UseCheckpoint, err = flags.GetBool(flagUseCheckpoint)
+	if err != nil {
+		return errors.Trace(err)
+	}
 	gcTTL, err := flags.GetInt64(flagGCTTL)
 	if err != nil {
 		return errors.Trace(err)
 	}
+	// if use checkpoint and gcTTL is the default value
+	// update gcttl to 1 hours
+	if cfg.UseCheckpoint && gcTTL == utils.DefaultBRGCSafePointTTL {
+		gcTTL = 3600
```

Comment:
maybe add some log here if we change the default configuration.

Reply:
done.